birdhouse/config/geoserver/service-config.json.template

```diff
@@ -1,7 +1,7 @@
 {
     "name": "geoserver",
     "url": "https://${PAVICS_FQDN_PUBLIC}/geoserver",
-    "type": ["geoserver"],
+    "type": ["data", "service-wms"],
```

Comment:
Not only a WMS, but also a WFS. 
(technically, WPS also possible, but not used)